content/en/continuous_integration/guides/github_gating.md

```diff
@@ -0,0 +1,76 @@
+kind: guide
+description: Learn how to use Datadog monitors to perform quality checks prior to deploying your GitHub applications.
+further_reading:
+  - link: "https://www.datadoghq.com/blog/datadog-github-actions-ci-visibility/"
+    tag: "Blog"
+    text: "Monitor your GitHub Actions workflows with Datadog CI Visibility"
+  - link: "/integrations/guide/source-code-integration"
+    tag: "Documentation"
+    text: "Learn about the GitHub integration"
+  - link: "https://docs.datadoghq.com/continuous_integration/guides/pull_request_comments/"
+    tag: "Documentation"
+    text: "Enable test summaries on your GitHub pull requests"
+---
+
+## Overview
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+Datadog is an integration partner for GitHub Actions Deployment Protection Rules, which helps you ensure the delivery of high-quality applications to end customers. You can enforce quality gates on your GitHub Actions deployment workflows using Datadog monitors. 
+
+These features are available for all Datadog customers, and they do not require usage of CI Visibility.
+
+## Enabling Deployment Protection Rules
+In order to rely on Datadog for quality checks on application deployments, you must have the Deployment Protection Rules feature enabled for your application.
+
+**Setting up a new GitHub application in Datadog**
+Follow instructions here[1] to create a GitHub application that is automatically connected to Datadog. When following these instructions, check the Deployment Protection Rules box.
+
+{{< img src="ci/github_gates_new_app.png" alt="Datadog GitHub pull request comment preview" style="width:100%;">}}
+
+If you already have a GitHub application set up and connected to Datadog, you can find a link to enable Deployment Protection Rules via the in-app [GitHub integration tile][2].
+
+{{< img src="ci/github_gates_existing_app.png" alt="Datadog GitHub pull request comment preview" style="width:100%;">}}
+
+**Configuring Deployment Protection Rules in GitHub** 
+1. Enable read and write permissions for Deployments
+2. Enable read permissions for Actions 
+3. Under “Subscribe to events” in an application, check “Deployment protection rule”
+4. In a repository, click “Settings”. Under the *Code and Automation* section, click *Environments*. Under *Deployment Protection Rules*, enable the GitHub application that is linked with Datadog integration. 
+
+## Creating monitors to gate your deployments
+Follow instructions [here][3] to create and configure a Datadog monitor to be used for GitHub Actions deployment gating. You can use multiple separate monitors for quality checks, but [composite monitors][4] are recommended, as they allow you to gate deployments based on two or more signals with one monitor. See all monitor types [here][5]. 
```

Comment:
```suggestion
## Create monitors to gate your deployments

Follow [these instructions][3] to create and configure a Datadog monitor to be used for GitHub Actions deployment gating. You can use multiple separate monitors for quality checks, but Datadog recommends  [composite monitors][4] because they allow you to gate deployments based on two or more signals with one monitor. For more information, see [Monitor Types][5]. 
```